src/System.CommandLine.Benchmarks/CommandLine/Perf_Parser_DefaultCtor.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.CommandLine.Parsing;
 using BenchmarkDotNet.Attributes;
 
 namespace System.CommandLine.Benchmarks.CommandLine
 {
     /// <summary>
-    /// Measures the performance of <see cref="System.CommandLine.Parser"/> default constructor.
+    /// Measures the performance of <see cref="Parsing.Parser"/> default constructor.
```

Comment:
Shouldn't this just be
```suggestion
    /// Measures the performance of <see cref="Parser"/> default constructor.
```